pennylane/pauli/utils.py

```diff
@@ -574,31 +595,10 @@ def pauli_word_to_matrix(pauli_word, wire_map=None):
     if wire_map is None:
         wire_map = {pauli_word.wires.labels[i]: i for i in range(len(pauli_word.wires))}
 
-    n_qubits = len(wire_map)
-
-    # If there is only a single qubit, we can return the matrix directly
-    if n_qubits == 1:
-        return pauli_word.matrix()
-
-    # There may be more than one qubit in the Pauli but still only
-    # one of them with anything acting on it, so take that into account
-    pauli_names = [pauli_word.name] if isinstance(pauli_word.name, str) else pauli_word.name
-
-    # Special case: the identity Pauli
-    if pauli_names == ["Identity"]:
-        return np.eye(2**n_qubits)
-
-    # If there is more than one qubit, we must go through the wire map wire
-    # by wire and pick out the relevant matrices
-    pauli_mats = [ID_MAT for x in range(n_qubits)]
-
-    for wire_label, wire_idx in wire_map.items():
-        if wire_label in pauli_word.wires.labels:
-            op_idx = pauli_word.wires.labels.index(wire_label)
-            # compute_matrix() only works because we work with Paulis here
-            pauli_mats[wire_idx] = getattr(qml, pauli_names[op_idx]).compute_matrix()
+    # pauli_word = pauli_word.map_wires(wire_map)
+    pr = pauli_word.pauli_rep
 
-    return reduce(np.kron, pauli_mats)
+    return pr.to_mat(wire_order=wire_map)
```

Comment:
Do we even need to convert it to a pauli rep?  We should be able to just do `pauli_word.matrix()`.

Reply:
I think the terminology is just confusing. The `pauli_word` is assumed to be an operator instance containing only paulis

Reply:
Ah you mean `Operator.matrix()`? That should work to, yes!

Reply:
Yep works, though is the pauli method not faster?

Reply:
✅